Repository/UnitRepository.php

```diff
@@ -45,7 +46,7 @@ class UnitRepository
 
     private $loaded = false;
 
-    public function __construct($config, Translator $translator, PersistenceInterface $persistence, Security $security)
+    public function __construct($config, TranslatorInterface $translator, PersistenceInterface $persistence, Security $security)
```

Comment:
I think the bundle was only tested on Symfony 2.0 in "real" conditions, maybe Sf2 @translator service is an instance of Translator ?

The change seems to work on VCA, so it's ok for me.